resources/views/about/index.blade.php

```diff
@@ -0,0 +1,227 @@
+@extends('layouts.librenmsv1')
+
+@section('title', __('About'))
+
+@section('content')
+<div class="modal fade" id="git_log" tabindex="-1" role="dialog" aria-labelledby="git_log_label" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title" id="myModalLabel">@lang('Local git log')</h4>
+            </div>
+            <div class="modal-body">
+                <pre>{!! $git_log !!}</pre>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-default" data-dismiss="modal">@Lang('Close')</button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-md-6">
+
+            <h3>@lang('LibreNMS is an autodiscovering PHP/MySQL-based network monitoring system')</h3>
+            <table class='table table-condensed table-hover'>
+                <tr>
+                    <td><b>@lang('Version')</b></td>
+                    <td><a href='http://www.librenms.org/changelog.html'>{{ $version_local }} - <span id='version_date'>{{ $git_date }}</span></a></td>
+                </tr>
+                <tr>
+                    <td><b>@lang('Database Schema')</b></td>
+                    <td>{{ $db_schema }}</td>
+                </tr>
+                <tr>
+                    <td><b>@lang('Web Server')</b></td>
+                    <td>{{ $version_webserver }}</td>
+                </tr>
+                <tr>
+                    <td><b>@lang('PHP')</b></td>
+                    <td>{{ $version_php }}</td>
+                </tr>
+                <tr>
+                    <td><b>@lang('MySQL')</b></td>
+                    <td>{{ $version_mysql }}</td>
+                </tr>
+                <tr>
+                    <td><b>@lang('RRDtool')</b></td>
+                    <td>{{ $version_rrdtool }}</td>
+                </tr>
+            </table>
+
+            <p>
+            <h3>@lang('LibreNMS is a community-based project')</h3>
+            @lang('Please feel free to join us and contribute code, documentation, and bug reports:')
+            <br>
+              <a href="http://www.librenms.org/">Web site</a> |
```

Comment:
Translations missing for these links (Git log should probably be "Local git log" too)